solr/core/src/test/org/apache/solr/update/processor/UUIDUpdateProcessorFallbackTest.java

```diff
@@ -165,7 +165,7 @@ SolrInputField field(String name, float boost, Object... values) {
 
   /** Convenience method for building up SolrInputFields with default boost */
```

Comment:
```suggestion
  /** Convenience method for building up SolrInputFields */
```

Does this mean then that `f` can be removed in favour of `field` actually?

Reply:
Can you elaborate?   as in make `f` be `field`?